c10/cuda/CMakeLists.txt

```diff
@@ -49,9 +49,7 @@ if(${COMPILER_SUPPORTS_HIDDEN_VISIBILITY})
 endif()
 
 # ---[ Dependency of c10_cuda
-target_link_libraries(c10_cuda PUBLIC c10)
-
-target_link_libraries(c10_cuda INTERFACE torch::cudart)
+target_link_libraries(c10_cuda PUBLIC c10 torch::cudart)
```

Comment:
Because `c10_cuda` does need to linked against cudart and it's a quirk of FindCUDA that this wasn't necessary before (it was linked to cudart automatically).